src/main/java/com/revature/models/Cohort.java

```diff
@@ -45,7 +45,7 @@
 	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
 	private LocalDate endDate;
 
-	@JsonIgnore
+//	@JsonIgnore
```

Comment:
Why was this @JsonIgnore removed? Was it necessary for your code?

If it is necessary then delete the comment. If not, uncomment it and let stay in.